extension.json

```diff
@@ -1,19 +1,20 @@
 {
 	"name": "Semantic Glossary",
-	"version": "4.0.0",
+	"version": "4.1.0",
```

Comment:
I'd go for 5.0.0 as we're dropping support for older versions and the requirements for Lingo have changed.